tests/test_dark_current.py

```diff
@@ -0,0 +1,433 @@
+"""
```

Comment:
Some of these tests are MIRI-specific; should they or the MIRI-specific functions they call be in the JWST repo instead of here in STCAL ? How strictly do we want to adhere to having no mission-specific code in STCAL ?